opencrvs-default.properties

```diff
@@ -74,8 +76,11 @@ MIDSCHEMAURL=${mosip.kernel.syncdata-service-idschema-url}
 SYNCSERVICE=${mosip.regproc.status.service.url}/registrationprocessor/v1/registrationstatus/sync
 PACKETRECEIVER=${mosip.packet.receiver.url}/registrationprocessor/v1/packetreceiver/registrationpackets
 RIDGENERATION=http://ridgenerator.kernel/v1/ridgenerator/generate/rid
+KEYMANAGER_TOKENID=${mosip.kernel.keymanager.url}/v1/keymanager/tokenid
+IDREPO_IDENTITY=${mosip.idrepo.identity.url}/idrepository/v1/identity/
+IDREPO_VID=${mosip.idrepo.vid.url}/idrepository/v1/vid
 
-IDSchema.Version=0.1
+IDSchema.Version=0.0
```

Comment:
Is this change correct ? which application uses this file ?

Reply:
@ase-101 to check on this

Reply:
waiting for OpenCRVS team

Reply:
removed the changes